packages/next-templates/src/components/TopTask/TopTaskCard.css

```diff
@@ -0,0 +1,13 @@
+.example-toptask-card {
+  padding: var(--voorbeeld-space-inline-rabbit);
+  background-color: var(--voorbeeld-color-violet-200);
+  --voorbeeld-toptask-inline-size-gaps-included: 550px;
+  block-size: 100%;
+  inline-size: calc(100% - var(--voorbeeld-toptask-inline-size-gaps-included));
+}
+
+.example-toptask-card__icon .voorbeeld-icon-clock {
+  --utrecht-icon-color: var(--voorbeeld-color-violet-800);
+  --utrecht-icon-size: var(--voorbeeld-size-icon-3xl);
+  margin-inline-start: -4px;
+}
```

Comment:
Volgens mij is er wat onnodige code duplication met `globals.css`

Reply:
Geen code duplication kunnen vinden.